contracts/core/chainlink-oracle/v1/test/ChainLinkOracle.t.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import "./TestUtil.sol";
+import "../../../../test/TestUtil.sol";
```

Comment:
I'd prefer a remapping to a "global" `tests` directory, so we can avoid all of these ugly deep relative imports.